packages/headless/src/features/commerce/context/context-state.ts

```diff
@@ -0,0 +1,22 @@
+import {ContextParams} from '../../../api/commerce/commerce-api-params';
+
+export interface CommerceContextState {
+  trackingId: string;
+  language: string;
+  currency: string;
+  // TODO: Does it make sense for the clientId to be part of the context state?
+  clientId: string;
+  context: ContextParams; // TODO: Should this be spread in its parent instead?
+}
+
+export const getContextInitialState = (): CommerceContextState => ({
+  trackingId: '',
+  language: '',
+  currency: '',
+  clientId: '',
+  context: {
```

Comment:
I wonder why we even need the context > view part anymore if there's just url under there + it's mandatory for PLP 😁 

I think I made a comment to that effect in the excel file where the plp v2 API request body is described